### PR TITLE
replace eddsa dependency with eddsa-api plugin

### PR DESCRIPTION
**Pull request by jtnord**
Now that sshd is not in core and a plugin this dependency seems to be incorrect here, however I am making the most minimal changes for plugins to use the eddsa-api and am not investing time in checking the hypothesis.

### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by jtnord** [comment]
needs Jenkins core bump 🤦

File: pom.xml [hunk 1]
```diff
@@ -32,16 +32,16 @@
...
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.361.x</artifactId>
-        <version>2102.v854b_fec19c92</version>
+        <artifactId>bom-2.426.x</artifactId>
+        <version>2555.v3190a_8a_c60c6</version>
```
Comment (jtnord) [inline pom.xml:44]: first release of the bom not the latest to reduce any uneeded bumps

**Comment by basil** [comment]
`subversion-plugin` is still shipping `eddsa-0.3.0.jar` as well.

**Comment by jtnord** [comment]
> `subversion-plugin` is still shipping `eddsa-0.3.0.jar` as well.

there is a quite a few plugins on the list to update.

**Comment by jtnord** [comment]
ping @jenkinsci/sshd-plugin-developers

**Pull request merged**